packages/rexray/extra/dcos-rexray.service

```diff
@@ -13,4 +13,6 @@ Environment=REXRAY_HOME=/
 ExecStartPre=/bin/rm -f /var/run/libstorage/lsx.lock
 # Allow REX-Ray configuration to be changed during upgrade.
 ExecStartPre=/bin/cp /opt/mesosphere/etc/rexray.conf /etc/rexray/config.yml
+# Prevent startup error: "open /run/docker/plugins/rexray.sock: no such file or directory"
+ExecStartPre=/bin/mkdir -p /run/docker/plugins
```

Comment:
This `mkdir -p` is apparently what older versions of REX-Ray did implicitly: https://github.com/rexray/rexray/issues/1108#issuecomment-343538897

It looks like I could also accomplish this by setting `CSI_ENDPOINT` to a path that exists (maybe `/run/dcos/rexray.sock`), but I'm not sure if that's a worse hack.

Reply:
Is this because there's a race between docker and RR startup?

Reply:
What I saw on CF clusters is that this directory was never created, blocking REX-Ray from starting. I'm guessing it's because they're running an older version of Docker. If it were a race between REX-Ray and Docker then I don't think there'd be a problem; REX-Ray would just fail and restart until Docker created `/run/docker/plugins/`.